podman system df -v crashes on unnamed images

<!--
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST

**NOTE** A large number of issues reported against Podman are often found to already be fixed
in more current versions of the project.  Before reporting an issue, please verify the
version you are running with `podman version` and compare it to the latest release
documented on the top of Podman's [README.md](../README.md).  If they differ, please
update your version of Podman to the latest possible and retry your command before creating
an issue.

If you are filing a bug against `podman build`, please instead file a bug
against Buildah (https://github.com/containers/buildah/issues). Podman build
executes Buildah to perform container builds, and as such the Buildah
maintainers are best equipped to handle these bugs.
-->

**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

<!--
Briefly describe the problem you are having in a few paragraphs.
-->
Running verbose version of `podman system df` crashes the software.

**Steps to reproduce the issue:**

1. Run `podman system df -v`

**Describe the results you received:**
The following trace is printed:
```
panic: runtime error: index out of range

goroutine 1 [running]:
main.getImageVerboseDiskUsage(0x1b479e0, 0xc0000bc018, 0xc000130e00, 0x3a, 0x40, 0xc0002aac00, 0x0, 0x0, 0xc0002aabd0, 0xc000289c08, ...)
	cmd/podman/system_df.go:467 +0xdc6
main.imagesVerboseOutput(0x1b479e0, 0xc0000bc018, 0xc000130e00, 0x3a, 0x40, 0x2af9b78, 0x0, 0x0, 0xc0002aabd0, 0xc0002aac00, ...)
	cmd/podman/system_df.go:552 +0x33a
main.verboseOutput(0x1b479e0, 0xc0000bc018, 0xc000130e00, 0x3a, 0x40, 0x2af9b78, 0x0, 0x0, 0xc0002aabd0, 0xc0002aac00, ...)
	cmd/podman/system_df.go:604 +0x7a
main.dfSystemCmd(0x2ad6540, 0x0, 0x0)
	cmd/podman/system_df.go:119 +0x1b2
main.glob..func100(0x2a77f20, 0xc0002b66d0, 0x0, 0x1, 0x0, 0x0)
	cmd/podman/system_df.go:40 +0x62
github.com/spf13/cobra.(*Command).execute(0x2a77f20, 0xc0000c4070, 0x1, 0x1, 0x2a77f20, 0xc0000c4070)
	vendor/github.com/spf13/cobra/command.go:826 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0x2a74aa0, 0xc0000aacf0, 0x7ffe9a91687b, 0x6)
	vendor/github.com/spf13/cobra/command.go:914 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
	vendor/github.com/spf13/cobra/command.go:864
main.main()
	cmd/podman/main.go:148 +0x88

```

**Describe the results you expected:**
podman not to crash and display detailed disk usage.

**Additional information you deem important (e.g. issue happens only occasionally):**
Non-verbose version of `podman system df` succeeds:
```
TYPE            TOTAL   ACTIVE   SIZE     RECLAIMABLE
Images          58      0        4.71GB   4.71GB (100%)
Containers      0       0        0B       0B (0%)
Local Volumes   0       0        0B       0B (0%)
```

**Output of `podman version`:**

```
Version:            1.4.5-dev
RemoteAPI Version:  1
Go Version:         go1.12.4
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  git commit: ""
  go version: go1.12.4
  podman version: 1.4.5-dev
host:
  BuildahVersion: 1.9.2
  Conmon:
    package: Unknown
    path: /usr/libexec/podman/conmon
    version: 'conmon version 1.0.0-dev, commit: 6f177c68ed989a6cc14ddbf8cec421819bb46401'
  Distribution:
    distribution: ubuntu
    version: "19.04"
  MemFree: 1175191552
  MemTotal: 8300961792
  OCIRuntime:
    package: 'cri-o-runc: /usr/bin/runc'
    path: /usr/bin/runc
    version: 'runc version spec: 1.0.1-dev'
  SwapFree: 2142236672
  SwapTotal: 2147479552
  arch: amd64
  cpus: 4
  hostname: eric-ubuntu
  kernel: 5.0.0-16-generic
  os: linux
  rootless: true
  uptime: 1h 55m 35.05s (Approximately 0.04 days)
registries:
  blocked: null
  insecure: null
  search: null
store:
  ConfigFile: /home/eric/.config/containers/storage.conf
  ContainerStore:
    number: 0
  GraphDriverName: vfs
  GraphOptions: null
  GraphRoot: /home/eric/.local/share/containers/storage
  GraphStatus: {}
  ImageStore:
    number: 0
  RunRoot: /tmp/1000
  VolumePath: /home/eric/.local/share/containers/storage/volumes
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Physical Ubuntu box.

--- a/cmd/podman/system_df.go
+++ b/cmd/podman/system_df.go
@@ -460,11 +460,11 @@ func getImageVerboseDiskUsage(ctx context.Context, images []*image.Image, images
 		}
 		var repo string
 		var tag string
-		if len(img.Names()) == 0 {
-			repo = "<none>"
-			tag = "<none>"
+		var repotags []string
+		if len(img.Names()) != 0 {
+			repotags = []string{img.Names()[0]}
 		}
-		repopairs, err := image.ReposToMap([]string{img.Names()[0]})
+		repopairs, err := image.ReposToMap(repotags)
 		if err != nil {
 			logrus.Errorf("error finding tag/digest for %s", img.ID())
 		}

